module/zfs/vdev_label.c

```diff
@@ -1161,6 +1230,14 @@ vdev_uberblock_sync(zio_t *zio, uberblock_t *ub, vdev_t *vd, int flags)
 	if (!vdev_writeable(vd))
 		return;
 
+	/* If the vdev was expanded, need to copy uberblock rings. */
+	if (vd->vdev_ops->vdev_op_leaf &&
```

Comment:
The `vd->vdev_ops->vdev_op_leaf` check here is redundant, the same check has already been made a few lines above.